pkgs/development/python-modules/pylint-odoo/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  pylint-plugin-utils,
+  pytestCheckHook,
+  pythonOlder,
+  pythonRelaxDepsHook,
+  setuptools,
+  validators,
+}:
+buildPythonPackage rec {
+  pname = "pylint-odoo";
+  version = "9.1.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "OCA";
+    repo = "pylint-odoo";
+    rev = "v${version}";
+    hash = "sha256-MXhQO7TsqL+rufI531nHiw4YZhJqZZFk3SAcGk0tl+g=";
+  };
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
```

Comment:
```suggestion
```
We no longer need to add `pythonRelaxDepsHook` explicitly.

> For ease of use, both buildPythonPackage and buildPythonApplication will automatically add pythonRelaxDepsHook if either pythonRelaxDeps or pythonRemoveDeps is specified.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook-using-pythonrelaxdepshook